core/src/main/java/org/apache/carbondata/core/datastore/chunk/reader/dimension/v2/CompressedDimensionChunkFileBasedReaderV2.java

```diff
@@ -121,6 +121,7 @@ public DimensionColumnPage decodeColumnPage(
     int[] invertedIndexesReverse = new int[0];
     int[] rlePage = null;
     DataChunk2 dimensionColumnChunk = null;
+    boolean isLongStringColumn = dimensionRawColumnChunk.isLongStringColumn();
```

Comment:
In V2 case it will be always false...as new encoder type will be only supported for V3 format

Reply:
The same as above. V2 does not support longStringColumn, so it is fine that the `isLongStringColumn` will always be false.

Reply:
The same as above